parser/model/ddl_test.go

```diff
@@ -50,7 +50,7 @@ func TestJobSize(t *testing.T) {
 - SubJob.ToProxyJob()
 `
 	job := model.Job{}
-	require.Equal(t, 320, int(unsafe.Sizeof(job)), msg)
+	require.Equal(t, 328, int(unsafe.Sizeof(job)), msg)
```

Comment:
What are 320 or 328 means?

Reply:
A new `int` member AdminCommandOperator has been added to struct `Job`, which increase the size of it to be with more 8 bytes.

The member AdminCommandOperator indicates where the command (cancel/pause/resume) on DDL jobs is from, end user or system. Refer to `parser.model.ddl.Job` .